master/docs/relnotes/index.rst

```diff
@@ -23,6 +23,8 @@ Features
 Fixes
 ~~~~~
 
+* Git: Respect the clobberOnFailure and retryFetch Git build step parameters when git fetch fails.
```

Comment:
`clobberOnFailure` and `retryFetch` are parameters and should be formatted as such.